modules/host/consts.go

```diff
@@ -53,11 +53,9 @@ var (
 	// furious for losing access to it for a few weeks.
 	defaultCollateralBudget = types.SiacoinPrecision.Mul64(100e3)
 
-	// defaultContractPrice defines the default price of creating a contract
-	// with the host. The default is set to 30 siacoins, which the file
-	// contract revision can have 15 siacoins put towards it, and the storage
-	// proof can have 15 siacoins put towards it.
-	defaultContractPrice = types.SiacoinPrecision.Mul64(3) // 3 siacoins
+	// with the host. The default is set to 0.05 siacoins and will be adjusted
+	// automatically if it needs to be higher
+	defaultContractPrice = types.SiacoinPrecision.Mul64(5).Div64(100) // 0.05 siacoins
```

Comment:
I think this defaultContractPrice is designed to cover transactionFees.  maybe they should be somehow tied?